index.html

```diff
@@ -231,10 +231,51 @@ <h2 class="section-tittle">Simple & Beautiful Interface</h2>
+                </div>
+                <div class="faq-box">
+                    <h5 class="faq-tittle">Can i get support from the Author?</h5>
+                    <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+                </div>
+                <div class="faq-box">
+                    <h5 class="faq-tittle">Do you have a free trail?</h5>
+                    <p>It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+                </div>
+            </div>
+            <div class="desine col-12 col-md-6">
+                <div class="faq-box">
+                    <h5 class="faq-tittle">How can I edit my personal information?</h5>
+                    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.</p>
+                </div>
+                <div class="faq-box">
+                    <h5 class="faq-tittle">Contact form isn't working?</h5>
+                    <p>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn't anything embarrassing hidden in the middle of text.</p>
+                </div>
+                <div class="faq-box">
+                    <h5 class="faq-tittle">Contact form isn't working?</h5>
+                    <p>The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text.</p>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="text-link col-6 ml-3">
```

Comment:
Labai susispaudzia prie siauro ekrano, gal is viso nereikia col, jei centruoja visada